Sources/PackageCollectionsSigning/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+# This source file is part of the Swift.org open source project
+#
+# Copyright (c) 2021 Apple Inc. and the Swift project authors
+# Licensed under Apache License v2.0 with Runtime Library Exception
+#
+# See http://swift.org/LICENSE.txt for license information
+# See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+
+add_library(PackageCollectionsSigning
+  Certificate/Certificate.swift
+  Key/BoringSSLKey.swift
+  Key/Key.swift
+  Key/Key+EC.swift
+  Key/Key+RSA.swift
+  Utilities/Utilities.swift)
+target_link_libraries(PackageCollectionsSigning PUBLIC
+  Crypto
+  CCryptoBoringSSL)
```

Comment:
should these be private instead of public? I think since we are using @_implementationOnly they should?

Reply:
amended https://github.com/apple/swift-package-manager/pull/3238/commits/2371ca0efca406ce0fc021f098bfa8f42296cae0